analysis_options.yaml

```diff
@@ -16,6 +16,17 @@ analyzer:
     deprecated_member_use: ignore
     deprecated_member_use_from_same_package: ignore
   exclude: # DIFFERENT FROM FLUTTER/FLUTTER
+    # Directories that might have Dart code, but aren't part of the engine, i.e.
+    # vended directories or code that is copied by tooling.
+    - prebuilts/
+    - third_party/
+    # Directories that have Dart code, but the Dart code isn't resolvable by the
```

Comment:
It looks like we already think we're analyzing some or all of this Dart code: https://github.com/flutter/engine/blob/main/ci/analyze.sh#L58 and below. Why are we not already hitting problems with this?